docs/src/solvers.md

```diff
@@ -95,3 +95,28 @@ JuMP.direct_model
 ```@docs
 JuMP.backend
 ```
+
+## Solver attributes
+
+Some solvers attributes can be queried through JuMP Models. Some solver attributes can setted through JuMP Models.
+
+```@docs
+solver_name
+```
+
+```@docs
+bridge_constraints
+```
+
+```@docs
+num_variables
```

Comment:
Is `num_variables` really a solver attribute? You can query it when no solver is attached.

Reply:
True, this could be in the variables section